res/css/views/right_panel/_UserInfo.scss

```diff
@@ -63,26 +63,30 @@ limitations under the License.
 
     .mx_UserInfo_avatar {
         margin: 24px 32px 0 32px;
-        cursor: pointer;
     }
 
     .mx_UserInfo_avatar > div {
         max-width: 30vh;
         margin: 0 auto;
     }
 
-    .mx_UserInfo_avatar > div > div {
+    .mx_UserInfo_avatar > div * {
         /* use padding-top instead of height to make this element square,
         as the % in padding is a % of the width (including margin,
         that's why we had to put the margin to center on a parent div),
         and not a % of the parent height. */
-        padding-top: 100%;
-        height: 0;
+        width: 30vh;
+        height: 30vh;
```

Comment:
Does the avatar still grow with the width of the right panel with this?

Reply:
Oh right I didn't notice it grew between a min and max, interesting

Reply:
Yeah, that's why I didn't go with MemberAvatar at the time, although that doesn't mean there isn't a way :)

Reply:
Arg I'm getting close 
![image](https://user-images.githubusercontent.com/2403652/71172185-7b0a6780-2257-11ea-8591-cae193e8ec72.png)